src/PoshGitTypes.ps1

```diff
@@ -141,12 +141,12 @@ class PoshGitTextSpan {
         }
         else {
             $bg = $this.BackgroundColor
-            if ($bg -and !(Test-VirtualTerminalSequece $bg)) {
+            if (($null -ne $bg) -and !(Test-VirtualTerminalSequece $bg)) {
```

Comment:
FYI this allows you to set the value to `0` (black).  Before, it would see the `0` as false in this condition and use the "default" color.